docs/source/getting-started.md

```diff
@@ -499,7 +532,12 @@ Each user may have one of these files in `$HOME/.jupyter/`.
 Jupyter also supports loading system-wide config files from `/etc/jupyter/`,
 which is the place to put configuration that you want to affect all of your users.
 
-## External services
+## External services (Change in version 0.8 for using OAuth2)
+
+Beginning with version 0.8, JupyterHub is also an OAuth2 provider. This allows
+JupyterHub to provide tokens for the Service to use, as well as allow the
+client to access and authenticate with the REST API on behalf of the
+Service. TODO How does this impact the next few paragraphs?
```

Comment:
The only change for external services related to OAuth is environment variables. There is now a JUPYTERHUB_CLIENT_ID environment variable to set the JupyterHub OAuth client id of the service.

The rest should be unchanged. The only bit that OAuth affects is how services may identify the originating user of an incoming request, which isn't discussed in this section.